packages/hash/api/src/model/entity.model.ts

```diff
@@ -614,25 +614,30 @@ class __Entity {
         })),
       );
 
-      await Promise.all(
-        entities.map(({ link, entity }) => {
-          if (link) {
-            const parentEntity = entities[link.parentIndex];
-            if (!parentEntity) {
-              throw new ApolloError("Could not find parent entity");
-            }
-            // links are created as an outgoing link from the parent entity to the children.
-            return parentEntity.entity.createOutgoingLink(client, {
-              createdByAccountId: user.accountId,
-              destination: entity,
-              stringifiedPath: link.meta.path,
-              index: link.meta.index ?? undefined,
-            });
-          } else {
-            return null;
+      /**
+       * Currently the outgoing links of an entity cannot be reliably created in parallel,
+       * because when indexed links are created the affected indexed links which have the same path
+       * and source entity have to be udpated. Running these operations in parallel can result in the
```

Comment:
```suggestion
       * and source entity have to be updated. Running these operations in parallel can result in the
```

Reply:
Addressed in https://github.com/hashintel/hash/pull/403/commits/21c179e2dee512f7cc7121f626af18ca745f7d65